deps/rabbit/src/rabbit_fifo.erl

```diff
@@ -1030,6 +1029,18 @@ handle_aux(leader, cast, {#return{msg_ids = MsgIds,
         _ ->
             {no_reply, Aux0, Log0}
     end;
+handle_aux(leader, _, {handle_tick, Fun, Args},
+           #?AUX{tick_pid = Pid} = Aux, Log, _) ->
+    NewPid =
+        case Pid =:= undefined orelse is_process_alive(Pid) =:= false of
```

Comment:
Suggestion, avoid inline boolean logic by introducing a new local function`process_is_alive/1` that handles the undefined case and spawn a process only if this function returns false.

Reply:
Got it, will do.